### DISCUSSION TITLE
关于宝塔面板使用alist播放本地挂载的视频时的问题

### DISCUSSION THREAD

**Discussion by Huayuluoshi** [AlistGo/alist#7201] [Q&A]
如题，我印象中开启反向代理后需要在nginx配置文件里添加什么东西来着，不然播放视频会先把视频缓存到本地再播放。请问哪位记得在哪个文件添加什么内容？感谢各位帮助！

**Comment by skiven2017** [comment]
在/www/server/nginx/conf/proxy.conf中或对应网站配置文件中设置禁用Nginx缓存，否则默认配置下访问较大文件时Nginx会先尝试将远程文件缓存至本机，导致播放失败
proxy_cache cache_one; # 删除这一行
proxy_max_temp_file_size 0; #加上这一行
